packages/itwinui-css/backstop/tests/button.html

```diff
@@ -1470,21 +1875,30 @@ <h3>Input + button combo group</h3>
     <section id="demo-combo-group">
       <div class="iui-button-group">
         <div>
-          <button class="iui-button iui-default"><span class="iui-button-label">Button 1</span></button>
+          <button
```

Comment:
HTML for these four buttons could be cleaned up as well

Reply:
Fixed.